src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -602,25 +601,20 @@ export class ActivityFillGraphQuerier {
    * Determine which samples contributed to a given height at a specific time. The result
    * is an array of all candidate samples, with their contribution amount.
    */
-  _getSamplesAtTime(time: number): $ReadOnlyArray<SampleContributionToPixel> {
-    const {
-      rangeStart,
-      rangeEnd,
-      treeOrderSampleComparator,
-      canvasPixelWidth,
-    } = this.renderedComponentSettings;
+  _getSamplesAtTime(
+    time: Milliseconds
+  ): $ReadOnlyArray<SampleContributionToPixel> {
+    const { rangeStart, treeOrderSampleComparator, xPixelsPerMs } =
+      this.renderedComponentSettings;
 
-    const rangeLength = rangeEnd - rangeStart;
-    const xPixelsPerMs = canvasPixelWidth / rangeLength;
```

Comment:
I've also removed these calculations. We have a `canvasPixelWidth` inside `renderedComponentSettings` already which is identical to this one./